content/ja/docs/concepts/overview/kubernetes-api.md

```diff
@@ -67,6 +69,68 @@ Kubernetes APIサーバーは、`/openapi/v2`エンドポイントを介してOp
 
 Kubernetesは、他の手段として主にクラスター間の連携用途向けのAPIに、Protocol buffersをベースにしたシリアライズフォーマットを実装しています。このフォーマットに関しては、[Kubernetes Protobuf serialization](https://github.com/kubernetes/design-proposals-archive/blob/main/api-machinery/protobuf.md)デザイン提案を参照してください。また、各スキーマのInterface Definition Language（IDL）ファイルは、APIオブジェクトを定義しているGoパッケージ内に配置されています。
 
+### OpenAPI V3
+
+{{< feature-state state="beta"  for_k8s_version="v1.24" >}}
+
+Kubernetes {{< param "version" >}} では、 OpenAPI v3によるAPI仕様をベータサポートとして提供しています。これは、デフォルトで有効化されているベータ機能です。kube-apiserverの`OpenAPIV3`という[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)を切ることにより、このベータ機能を無効化することができます。
+
+`/openapi/v3` が、全ての利用可能なグループやバージョンの一覧を閲覧するためのディスカバリーエンドポイントとして提供されています。 このエンドポイントは、JSONのみを返却します。利用可能なグループやバージョンは、 次のような形式で提供されます。
+
+```yaml
+{
+    "paths": {
+        ...,
+        "api/v1": {
+            "serverRelativeURL": "/openapi/v3/api/v1?hash=CC0E9BFD992D8C59AEC98A1E2336F899E8318D3CF4C68944C3DEC640AF5AB52D864AC50DAA8D145B3494F75FA3CFF939FCBDDA431DAD3CA79738B297795818CF"
+        },
+        "apis/admissionregistration.k8s.io/v1": {
+            "serverRelativeURL": "/openapi/v3/apis/admissionregistration.k8s.io/v1?hash=E19CC93A116982CE5422FC42B590A8AFAD92CDE9AE4D59B5CAAD568F083AD07946E6CB5817531680BCE6E215C16973CD39003B0425F3477CFD854E89A9DB6597"
+        },
+        ....
+    }
+}
+```
+<!-- for editors: intionally use yaml instead of json here, to prevent syntax highlight error. -->
+
+クライアントサイドのキャッシングを改善するために、相対URLはイミュータブルなOpenAPI記述を指しています。
+また、APIサーバーも、同様の目的で適切なHTTPキャッシュヘッダー（`Expires`には1年先の日付、`Cache-Control`には`immutable`）をセットします。廃止されたURLが使用された場合、APIサーバーは最新のURLへリダイレクトを行います。
+
+Kubernetes APIサーバーは、`/openapi/v3/apis/<group>/<version>?hash=<hash>`のエンドポイントにて、KubernetesのグループバージョンごとにOpenAPI v3仕様を公開しています。
+
+受理されるリクエストヘッダーについては、以下の表の通りです。
+
+<table>
+  <caption style="display:none">OpenAPI v3において有効なリクエストヘッダー</caption>
+  <thead>
+     <tr>
+        <th>ヘッダー</th>
```

Comment:
If translate "Header", "Possible values", and "Notes", it's better to translate the table in "OpenAPI V2" section, too.

Reply:
Are you going to translate these table headers in "OpenAPI V2" section?
I'd like it to match either English or Japanese.

Reply:
fixed in [4f36506](https://github.com/kubernetes/website/pull/37008/commits/4f36506847a1fd17ad48c78efa16abecc08ea922)

Reply:
honestly, I don't have a strong preference, but as you said, It would be more consistent with translating table headers.